frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.stories.tsx

```diff
@@ -35,6 +37,9 @@ export default {
     WaitForResizeToStopDecorator,
     MockIsEmbeddingDecorator,
   ],
+  parameters: {
+    layout: "fullscreen",
+  },
```

Comment:
It seems when loki runs the test it's in this mode, this is to ensure we see the same result as when loki takes snapshots.